sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/rest/SormasToSormasRestClient.java

```diff
@@ -40,28 +40,37 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import de.symeda.sormas.api.SormasToSormasConfig;
 import de.symeda.sormas.api.i18n.I18nProperties;
 import de.symeda.sormas.api.i18n.Strings;
 import de.symeda.sormas.api.sormastosormas.SormasToSormasEncryptedDataDto;
 import de.symeda.sormas.api.sormastosormas.SormasToSormasErrorResponse;
 import de.symeda.sormas.api.sormastosormas.SormasToSormasException;
 import de.symeda.sormas.api.sormastosormas.ValidationErrors;
-import de.symeda.sormas.backend.common.StartupShutdownService;
+import de.symeda.sormas.backend.sormastosormas.SormasToSormasEncryptionFacadeEjb.SormasToSormasEncryptionFacadeEjbLocal;
+import de.symeda.sormas.backend.sormastosormas.access.SormasServerReference;
+import de.symeda.sormas.backend.sormastosormas.access.SormasToSormasDiscoveryService;
+import de.symeda.sormas.backend.sormastosormas.rest.auth.Oidc;
 import de.symeda.sormas.backend.util.ClientHelper;
 
 public class SormasToSormasRestClient {
```

Comment:
Let's add this comment here:
```
Meant to be instantiated using SormasToSormasRestClientProducer.
We are not using a LocalBean here, because we need to be able to mock this class for unit testing.
Alternatively, we could use Apache DeltaSpike: https://deltaspike.apache.org/documentation/test-control.html#MockFrameworks
```

Reply:
comment added